features/prompt-registry.mdx

```diff
@@ -121,12 +129,35 @@ promptlayer.prompts.publish(
     prompt_template=pt,
 )
 ```
-They are also available in the following REST API endpoints 
+
+They are also available in the following REST API endpoints
 
 `/rest/get-prompt-template` ([read more](/reference/get-prompt-template))
 
 `/rest/publish-prompt-template` ([read more](/reference/publish-prompt-template))
 
+### Metadata
+
+Custom metadata can be associated with individual prompt templates versions. This allows you to set the provider, model, temperature... etc for each prompt template. It is important to note here that `model` attribute is reserved for the model parameters.
```

Comment:
"individual prompt template versions" not "individual prompt templates versions"

Reply:
"This allows you to set values such as provider, model, or temperature for each prompt template."

Not

"This allows you to set the provider, model, temperature... etc for each prompt template."

Reply:
Updated